Vegeting/Screens/CreateGroup/ViewController/SecondCreateGroupViewController.swift

```diff
@@ -224,9 +320,70 @@ final class SecondCreateGroupViewController: BaseViewController {
         }
     }
     
+    override func configureUI() {
+        scrollView.showsVerticalScrollIndicator = false
+        view.backgroundColor = .systemBackground
+    }
+    
+    private func applyEditingTextViewForm() {
+         contentTextView.backgroundColor = .systemBackground
+         contentTextView.layer.borderColor = UIColor.vfYellow1.cgColor
+         contentTextView.layer.borderWidth = 1.5
+     }
+
+     private func applyEndEditingTextViewForm() {
+         contentTextView.backgroundColor = .vfGray4
+         contentTextView.layer.borderColor = UIColor.vfYellow1.cgColor
+         contentTextView.layer.borderWidth = 0
+     }
+    
+    private func applyEditingTextFieldForm() {
+        titleTextField.backgroundColor = .systemBackground
+        titleTextField.layer.borderColor = UIColor.vfYellow1.cgColor
+        titleTextField.layer.borderWidth = 1.5
+    }
+
+     private func applyEndEditingTextFieldForm() {
+         titleTextField.backgroundColor = .vfGray4
+         titleTextField.layer.borderColor = UIColor.vfYellow1.cgColor
+         titleTextField.layer.borderWidth = 0
+     }
+    
+    private func observeKeyboardWillShow() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(scrollVerticalWhenKeybaordWillShow),
+                                               name: UIResponder.keyboardWillShowNotification,
+                                               object: nil)
+    }
+    
+    @objc
+    private func scrollVerticalWhenKeybaordWillShow(notification: NSNotification) {
+        let bottomOffset = CGPoint(x: 0, y: scrollView.contentSize.height - scrollView.bounds.size.height + scrollView.contentInset.bottom)
+        if bottomOffset.y > 0 {
+            scrollView.setContentOffset(bottomOffset, animated: true)
```

Comment:
p3;
해당 offset의 의미는 bottomOffset이라기 보단, 현재 scrollView offset + 해당 offSet = ScrollView의 bottomOffset인 것 같습니다.
그래서 bottomOffset대신 scrollHeightOffset이라는 표현은 어떨까요? 또한 y를 계산한 식들을 따로 빼서 scrollHeight라는 점을 좀 더 부각하면 가독성이 더 좋을 것 같습니다. 

```suggestion
        let scrollHeight =  scrollView.contentSize.height - scrollView.bounds.size.height
        let scrollHeightOffset = CGPoint(x: 0, y: scrollHeight)
        if scrollHeight > 0 {
            scrollView.setContentOffset(bottomOffset, animated: true)
```

Reply:
[cae14c9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/165/commits/cae14c954e2838b0d9c18422cd31197d73298263) 요렇게 반영해봤는데 어떨까요?

Reply:
아주 좋숩니당